protocol/contracts/beanstalk/AppStorage.sol

```diff
@@ -570,40 +464,27 @@ contract Storage {
  * @param plenty The amount of plenty token held by the contract.
  */
 struct AppStorage {
-    uint8 deprecated_index;
-    int8[32] deprecated_cases;
     bool paused; // ────────┐ 1
     uint128 pausedAt; // ───┘ 16 (17/32)
     Storage.Season season;
-    Storage.Contracts c;
-    Storage.Field f;
-    Storage.Governance g;
+    Storage.Field field;
     Storage.CurveMetapoolOracle co;
-    Storage.Rain r;
-    Storage.Silo s;
+    Storage.Rain rain;
+    Storage.Silo silo;
     uint256 reentrantStatus;
-    Storage.Weather w;
+    Storage.Weather weather;
     uint256 earnedBeans;
-    uint256[14] deprecated;
-    mapping(address => Account.State) a;
-    uint32 deprecated_bip0Start; // ─────┐ 4
-    uint32 deprecated_hotFix3Start; // ──┘ 4 (8/32)
-    mapping(uint32 => Storage.Deprecated_Fundraiser) deprecated_fundraisers;
-    uint32 deprecated_fundraiserIndex; // 4 (4/32)
-    mapping(address => bool) deprecated_isBudget;
+    mapping(address => Account.State) accountStates;
     mapping(uint256 => bytes32) podListings;
     mapping(bytes32 => uint256) podOrders;
     mapping(address => Storage.AssetSilo) siloBalances;
-    mapping(address => Storage.SiloSettings) ss;
-    uint256[2] deprecated2;
-    uint128 deprecated_newEarnedStalk; // ──────┐ 16
-    uint128 deprecated_vestingPeriodRoots; // ──┘ 16 (32/32)
+    mapping(address => Storage.SiloSettings) siloSettings;
     mapping(uint32 => uint256) sops;
     // Internal Balances
     mapping(address => mapping(IERC20 => uint256)) internalTokenBalance;
     // Unripe
     mapping(address => mapping(address => bool)) unripeClaimed;
-    mapping(address => Storage.UnripeSettings) u;
+    mapping(address => Storage.UnripeSettings) unripeSettings;
```

Comment:
natspec defines the param `u`, could we change?

Reply:
fixed in https://github.com/BeanstalkFarms/Beanstalk/commit/7ea86b8f732de1bd4bea2313c197882e15f888b6